### PR TITLE
Update menu and logo

### PR DESCRIPTION
**Pull request by jinety**

### CODE REVIEW

File: practice-two/src/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
```
Comment (phuocduong-agilityio) [inline practice-two/src/index.html]: What is this?

File: practice-two/src/index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                            <a href="" class="link">Customers</a>
+                        </li>
+                        <li class="menu-item">
+                            <a href="" class="link">Pricing</a>
+                        </li>
+                        <li class="menu-item">
+                            <a href="" class="link">Resources</a>
+                        </li>
+                    </ul>
+                        <button class="btn" type="button">Sign In</button>
```
Comment (phuocduong-agilityio) [inline practice-two/src/index.html]: Indent is wrong

File: practice-two/src/index.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                        <h1>Product</h1>
+                    </div>
+                </div>
+                <div class="navbar">
+                    <i class="fas fa-bars"></i>
+                </div>
+                <div class="menu-button">
+                    <ul class="menu">
+                        <li class="menu-item">
+                            <a href="" class="link">Product</a>
```
Comment (phuocduong-agilityio) [inline practice-two/src/index.html]: Don't use href with empty value, pls check comment in first practice

File: practice-two/src/style.css [hunk 4]
```diff
@@ -0,0 +1,201 @@
+/* http://meyerweb.com/eric/tools/css/reset/
```
Comment (phuocduong-agilityio) [inline practice-two/src/style.css]: Split reset to reset.css file. Btw, pls format reset css

File: practice-two/src/style.css [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+    --primary-color-200: #173A56;
+    --primary-color-300: #22343D;
+    --secondary-color-100: white;
+
+}
+
+/*START MAIN HEADER*/
+.main-header {
+    width: 100%;
+    min-height: 100vh;
```
Comment (phuocduong-agilityio) [inline practice-two/src/style.css]: Why do you need this style?

File: practice-two/src/style.css [hunk 6]
```diff
@@ -0,0 +1,201 @@
...
+    max-width: 100%;
+}
+
+.container {
+    display: flex;
+    width: 1200px;
+    max-width: 100%;
```
Comment (phuocduong-agilityio) [inline practice-two/src/style.css]: Why do you need this style?

File: practice-two/src/style.css [hunk 7]
```diff
@@ -0,0 +1,201 @@
...
+    display: none;
+    color: var(--primary-color-300);
+    text-align: right;
+    cursor: pointer;
+    padding: 27px;
+}
+
+.fas fa-bars {
+    width: auto;
+}
```
Comment (phuocduong-agilityio) [inline practice-two/src/style.css]: Missing empty line

File: practice-two/src/index.html (hunk 1 above)
Comment (jinety) [reply practice-two/src/index.html]: the link of the icon bar

Comment (phuocduong-agilityio) [reply practice-two/src/index.html]: Pls do it yourself, you just use this library for a simple icon. If we use this solution, our application will be heavily

File: practice-two/src/css/reset.css [hunk 8]
```diff
@@ -0,0 +1,48 @@
+/* http://meyerweb.com/eric/tools/css/reset/
```
Comment (phuocduong-agilityio) [inline practice-two/src/css/reset.css:1]: Format this file

File: practice-two/src/css/style.css [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+    --font-weight-m: 600;
+    --font-weight-l: 700;
+    /*FONT FAMILY*/
+    --font-family: 'Poppins', sans-serif;
+    /*COLOR*/
+    --primary-color-100: #02897A;
+    --primary-color-200: #173A56;
+    --primary-color-300: #22343D;
+    --secondary-color-100: white;
+
```
Comment (phuocduong-agilityio) [inline practice-two/src/css/style.css]: Remove empty line

File: practice-two/src/css/style.css [hunk 10]
```diff
@@ -0,0 +1,138 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@500;600;700&display=swap');
+:root {
+    /*FONT SIZE*/
```
Comment (phuocduong-agilityio) [inline practice-two/src/css/style.css]: `/*FONT SIZE*/` -> `/* FONT SIZE */` => Add space

File: practice-two/src/css/style.css [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+    --font-size-xs: 16px;
+    --font-size-xl: 30px;
```
Comment (phuocduong-agilityio) [inline practice-two/src/css/style.css]: After a block of variables, should add empty line

**Pull request merged**